axelor-account/src/main/java/com/axelor/apps/account/service/invoice/InvoiceTermServiceImpl.java

```diff
@@ -414,4 +417,18 @@ public boolean checkInvoiceTermDeletionConditions(Invoice invoice) {
     }
     return false;
   }
+
+  @Override
+  @Transactional(rollbackOn = {AxelorException.class, Exception.class})
+  public void refusalToPay(
```

Comment:
This method does not throw AxelorException, and Exception.class already includes every exception. Here we can keep only Exception or only write @Transactional since this method does not throw checked exception.